stable/mission-control/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: mission-control
 home: https://jfrog.com/mission-control/
-version: 4.0.1
-appVersion: 4.4.2
+version: 4.1.0
+appVersion: 4.5.0
 description: A Helm chart for JFrog Mission Control
 sources:
```

Comment:
@rahulsadanandan  can you bump it to 4.2.0 (we need to reserve 4.1.0 for 4.4.0 release)